google/datalab/contrib/pipeline/_pipeline.py

```diff
@@ -217,13 +218,47 @@ def _get_bq_execute_params(operator_task_details):
       operator_task_details['bql'] = operator_task_details['query'].sql
       del operator_task_details['query']
 
+    if 'parameters' in operator_task_details:
+      operator_task_details['query_params'] = Pipeline._get_query_parameters(
+        operator_task_details['parameters'])
+      del operator_task_details['parameters']
```

Comment:
It will be helpful if we document why these fields need to be removed from "operator_task_details". Perhaps add a "Returns: xxxxxx" section in "_get_operator_param_name_and_values" function?

Reply:
Done.